src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -629,6 +649,32 @@ public void setImplVelocity(Vector3d velocity) {
     }
 
     @Override
+    public Map<Statistic, Long> getStatistics() {
+        StatisticsFile statisticsFile = statsFile;
```

Comment:
I guess I did it to reduce the diff to SpongeUser's impl. But I changed that now.